rpcs3/Emu/System.cpp

```diff
@@ -1587,9 +1588,9 @@ game_boot_result Emulator::Load(const std::string& title_id, bool is_disc_patch,
 				vfs::mount("/dev_hdd0/game/" + m_title_id, game_dir + '/');
 			}
 		}
-		else if (m_cat == "DG" && from_hdd0_game && disc.empty())
+		else if (!inherited_ps3_game.empty() || (m_cat == "DG" && disc.empty()))
```

Comment:
Why remove the condition? We don't want people to just paste their games anywhere.

Reply:
It's HW correct, PS3_GAME is actually a virtual directory. LV2 disaaembly has even a specific check for it.

Reply:
I mean why remove from_hdd0_game check.  Where else would it be

Reply:
Anywhere. Like in /host_root. I discovered it by accident when testing.
host_root is the mount point of the host filesystem, the one which boots external software from remote PC to debug console for example.

Reply:
This also makes the else path obsolete.
I don't think we should remove it.
There is no reason to support installed games outside of hdd0

Reply:
What if I make a hw test and put it outside HDD0? why should the result differ from PS3?

Reply:
Because we are dealing with smoothbrains in the discord help section.

Reply:
You're literally the only one ever trying this.
Why would you expose such a potential desaster to everyone.

Reply:
Reverted.